client/src/index.js

```diff
@@ -33,6 +34,7 @@ const Routes = () => {
           <Route path="/about/" component={About} />
           <Route path="/status/" component={Status} />
         </div>
+        <Footer />
       </div>
     </Router>
   );
```

Comment:
can you add the package.json and commit it to this PR, material UI dependency is not in the master hence the deployment is failing.